src/classes/CON_PrimaryAffls_TEST.cls

```diff
@@ -768,51 +768,492 @@ private with sharing class CON_PrimaryAffls_TEST {
+
+        UTIL_CustomSettings_API.getSettingsForTests(new Hierarchy_Settings__c(Affiliation_Record_Type_Enforced__c = false));
+
+        String namespace = UTIL_Namespace.getNamespace();
+        String prefix = (String.isNotBlank(namespace)) ? namespace + '__' : '';
+
+        List<Affl_Mappings__c> mappings = new List<Affl_Mappings__c>();
+        mappings.add(new Affl_Mappings__c(Name = 'Business Organization', Account_Record_Type__c = 'Not a Valid RecordType', Primary_Affl_Field__c = prefix + 'Gender__c'));
+        mappings.add(new Affl_Mappings__c(Name = 'Household Account', Account_Record_Type__c = 'HH_Account', Primary_Affl_Field__c =  prefix + 'Primary_Household__c'));
+        insert mappings;
+
+        Test.startTest();
+        insert testContactsList;       
+        Test.stopTest();
+
+        List<Contact> testContactResultsList = [SELECT Id, Primary_Organization__c
+                                                FROM Contact 
+                                                WHERE Id IN :testContactsList];
+
+        System.assertEquals(100, testContactResultsList.size());
+    }
+
+    /*************************************************************************************************************
+    * @description Test case to verify that updating a primary affiliation field on Contact with record type
+    * validation disabled, a valid Contact field mapping to a non-lookup field and invalid Record Type mapping does not 
+    * throw an error when the mapped field is not populated.
+    **************************************************************************************************************/
+    @isTest 
+    private static void insertContactAfflInvalidRTAndInvalidFieldMappingFieldNotPopulatedPositive(){
+        List<Contact> testContactsList = UTIL_UnitTestData_TEST.getMultipleTestContacts(100);
+       
+        UTIL_CustomSettings_API.getSettingsForTests(new Hierarchy_Settings__c(Affiliation_Record_Type_Enforced__c = false));
+
+        String namespace = UTIL_Namespace.getNamespace();
+        String prefix = (String.isNotBlank(namespace)) ? namespace + '__' : '';
+
+        List<Affl_Mappings__c> mappings = new List<Affl_Mappings__c>();
+        mappings.add(new Affl_Mappings__c(Name = 'Business Organization', Account_Record_Type__c = 'Not a Valid RecordType', Primary_Affl_Field__c = prefix + 'Gender__c'));
+        mappings.add(new Affl_Mappings__c(Name = 'Household Account', Account_Record_Type__c = 'HH_Account', Primary_Affl_Field__c =  prefix + 'Primary_Household__c'));
+        insert mappings;
+
+        Test.startTest();
+            insert testContactsList;
+        Test.stopTest();
+
+        List<Contact> testContactResultsList = [SELECT Id, Primary_Organization__c
+                                                FROM Contact 
+                                                WHERE Id IN :testContactsList];
+
+        System.assertEquals(100, testContactResultsList.size());
```

Comment:
Do we want to validate expected field values here in a loop? It might add value.